assets/js/googleMaps.js

```diff
@@ -1,75 +0,0 @@
-
-//Create global variable to store the latitude and longitude for the user
-var userLatLang;
-
-
-//
-
-function initMap(items) {
-  userLatLang = {lat: userlat, lng: userlng};  
-  
-  var map = new google.maps.Map(document.getElementById("map_div"), {
-    zoom: 11,
-    center: userLatLang,
-    mapTypeId: google.maps.MapTypeId.ROADMAP
-  });
```

Comment:
Semicolons after function declarations are unnecessary. An assignment of a function expression to a variable does require semicolons. 

```
function initMap() {
} // No semicolon here

var initMap = function () {
}; // Semicolon belongs here
```

Keep in mind that statements require semicolons and that an assignment is a statement.